script/update-external-links.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-param-reassign */
+const cheerio = require('cheerio');
+
+function updateExternalLinks() {
+  return (files, metalsmith, done) => {
+    for (const fileName of Object.keys(files)) {
+      const file = files[fileName];
+      let linkUpdated = false;
+
+      if (fileName.endsWith('html')) {
+        const doc = cheerio.load(file.contents);
+        doc('a[href^="http"]').each((i, el) => {
+          const link = doc(el);
+          const relAttr = link.attr('rel');
+          const targetAttr = link.attr('target');
+          const hrefAttr = link.attr('href') || '';
+
+          // We want to make sure links that open in a new tab
+          // always have noopener, but data-allow-opener is an
+          // escape hatch if we need it
+          if (
+            targetAttr === '_blank' &&
+            !link.attr('data-allow-opener') &&
+            (!relAttr || !relAttr.includes('noopener'))
+          ) {
+            linkUpdated = true;
+            link.attr('rel', `noopener ${relAttr || ''}`);
+          }
+
+          // If a link makes it through this logic, it's an
+          // external link that should always open in a new tab
+          // There is an escape hatch here, too
+          if (
+            !hrefAttr.includes('va.gov') &&
+            !hrefAttr.includes('vets.gov') &&
+            !link.attr('data-no-external')
+          ) {
+            if (targetAttr !== '_blank') {
+              linkUpdated = true;
+              link.attr('target', '_blank');
+            }
+
+            if (!relAttr || !relAttr.includes('noopener')) {
+              linkUpdated = true;
+              link.attr('rel', `noopener ${relAttr || ''}`);
+            }
```

Comment:
Does this logic need to be here too? Looks pretty similar to line 24.

Reply:
It's the same logic, but this is run for links that should have `target="_blank"` but didn't, and on line 24 it's run on links that already had `target="_blank"`.